.deepsource.toml

```diff
@@ -0,0 +1,13 @@
+version = 1
+
+exclude_patterns = [
```

Comment:
we should exclude database directory as well because it generally requires no modification